app/controllers/vendor_api/decisions_controller.rb

```diff
@@ -1,9 +1,9 @@
 module VendorAPI
   class DecisionsController < VendorAPIController
     include ApplicationDataConcerns
+    include APIValidationsAndErrorHandling
```

Comment:
although we chatted about this in our meeting today and agreed to keep it, on revisiting I think Im keen on avoiding a refactor here as this is the existing API validation in v1.0, and I would prefer if we keep this refactor along with the others till the ticket in the backlog, mainly to avoid any unexpected behaviour/repercussions while we refine our validation concern. Let me know your thoughts

Reply:
Ok I can remove commit https://github.com/DFE-Digital/apply-for-teacher-training/pull/6416/commits/e17ece11ba0f4e8fef81049a116eb433650b3446 from this PR. I'll create a separate ticket to address the error message issue and a different one to revisit including the validations module.